src/game/client/worldheightmap.cpp

```diff
@@ -14,11 +14,27 @@
  */
 #include "worldheightmap.h"
 #include "file.h"
+#include "filesystem.h"
+#include "globaldata.h"
 #include "inputstream.h"
-#ifdef GAME_DLL
-#include "hooker.h"
+#include "mapobject.h"
+#include "polygontrigger.h"
+#include "sideslist.h"
+#include "staticnamekey.h"
+#include "terraintex.h"
+#include "terraintypes.h"
+#include "thingfactory.h"
+#include "w3dshadow.h"
+
+static float HEIGHTMAP_SCALE = 0.625f;

```

Comment:
This looks like its meant to be a constant but it is not marked as const. Suggest making this `static constexpr`

Reply:
fixed.